### DISCUSSION TITLE
valhalla/valhalla spam

### DISCUSSION THREAD

**Discussion by akyudumsu** [community/community#68299] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/45712012/54a9e446-6c1f-422e-b5b1-3c56f8661689)

![image](https://github.com/community/community/assets/45712012/8923a044-e14a-411f-89b0-3dfe24c289fe)

**Comment by OlivierJM** [comment]
I filed a similar complaint here https://github.com/orgs/community/discussions/68194 The whole Github homepage has become very unusable.

**Comment by Zaczero** [comment]
![image](https://github.com/community/community/assets/10835147/90230609-96cb-418e-baf5-2d6753ff54d9)

All of my "explore" repositories have already been starred by me quite a while ago. What's the point of having an "explore" feature that suggests projects I've already explored?

**Comment by Yakiyo** [comment]
same issue for me. have been seeing double valhalla and bitwardens clients repo in my explore tab consistently for a while. in other cases too, two out of the three recommendations are the same ones. as in theres always been a duplicate one for me

**Reply by OlivierJM** [reply]
It sounds as if they are experimenting showing repos that have paid, some kind of advertisment 🤔

**Comment by jeongukjae** [comment]
<img width="312" alt="스크린샷 2023-09-26 오후 6 37 01" src="https://github.com/community/community/assets/8815362/1e27373a-c133-4d33-8aaf-8b26bf9e118c">

Same here. I thought there's a bug in explore tab.
